Sources/ApodiniDatabase/Handlers/Create.swift

```diff
@@ -4,7 +4,7 @@ import Apodini
 /// A Handler that creates, if possible, an object in the database that conforms to `DatabaseModel` out of the body of the request.
 /// It uses the database that has been specified in the `DatabaseConfiguration`.
 public struct Create<Model: DatabaseModel>: Handler {
-    @Apodini.Environment(\.database)
+    @Apodini.Environment(\.db)
```

Comment:
I would suggest to keep the name `database` instead of `db`. I would suggest renaming the property name of the `Application`